applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -410,7 +405,6 @@ static void le_audio_msg_sub_thread(void)
 			ERR_CHK(ret);
 
 			break;
-
```

Comment:
I added blank lines the places where there was none in #10, so we have to agree on what to do. I think it helps readability to have the blank lines

Reply:
I added back the blank lines